frontend/src/scenes/session-recordings/components/InternalSurvey/InternalMultipleChoiceSurvey.tsx

```diff
@@ -0,0 +1,277 @@
+/**
+ * @fileoverview A component that displays an interactive survey within a session recording. It handles survey display, user responses, and submission
+ */
+import { LemonButton, LemonCheckbox, LemonTextArea, Link } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { maxThreadLogic } from 'scenes/max/maxThreadLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+
+import { SidePanelTab, SurveyQuestion, SurveyQuestionType } from '~/types'
+
+import { internalMultipleChoiceSurveyLogic } from './internalMultipleChoiceSurveyLogic'
+import MaxTool from 'scenes/max/MaxTool'
+import { maxLogic } from 'scenes/max/maxLogic'
+import { sidePanelSettingsLogic } from '~/layout/navigation-3000/sidepanel/panels/sidePanelSettingsLogic'
+
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+
+interface InternalSurveyProps {
+    surveyId: string
+}
+
+function TooExpensiveHelpMessage(): JSX.Element {
+    return (
+        <>
+            <p>
+                We aim to be significantly cheaper than our competitors. In addition to our pay-as-you-go pricing, below
+                are few ways to reduce costs by recording fewer sessions:
+                <ul className="list-disc pl-4 text-secondary">
+                    <li>Disable automatic recording and programmatically start and stop recordings</li>
+                    <li>
+                        Set a feature flag with the conditions that define which users or sessions you want to record.
+                    </li>
+                    <li>Set a minimum recording duration.</li>
+                    <li>Set a sampling rate for recordings.</li>
+                    <li>
+                        Make sure you capture at least one event per session to reduce the costs while maintaining all
+                        filtering options.
+                    </li>
```

Comment:
i'd leave this out from here

it's an implementation detail to avoid surprises but for many folks it's not necessary
we could even get fancy, detect it, and warn people in context instead of a passive warning

```suggestion
```